docs/support.md

```diff
@@ -0,0 +1,33 @@
+# Customer Support
+
+In context of CDN360 we consider 3 types of customers.
+
+## Customer Types
+
+1. **Trial Customer** - Customers who onboarded through the Web UI, but have not upgraded to paid customers. These customers have a Trial credit of $50. When they reach $50 consumption, their properties may be undeployed from production automatically. To continue working with the services, an upgrade to Paid Customer is required. No credit card is required to sign-up and start using the Trial account. 
```

Comment:
"paid customers" ,  "Paid Customer".   
Shouldn't we consistently use the defined term?
"sign-up" should be "sign up"

Reply:
I agree with Burton about capitalization and the use of singular/plural. Please let's be consistent. Here is my suggestion:
"upgraded to paid customer" = "upgraded to Paid Customer"
Also, I agree with Burton about making "sign-up" two words ("sign up").